ext/mini_racer_extension/mini_racer_extension.cc

```diff
@@ -57,6 +57,8 @@ typedef struct {
     ContextInfo* context_info;
     Local<String>* eval;
     useconds_t timeout;
+    size_t max_mem;
+    size_t current_mem;
```

Comment:
Yep. I'm familiar.
Luckily, EvalParams are not created in any significant manner. (1 per script run)
Unluckily for you, you're just going have to deal with knowing that it's staying there in the code. ;)